ipfix-flow-exporter/bpf_ipfix_egress_kern.c

```diff
@@ -305,7 +291,7 @@ static __always_inline bool parse_egress_eth(struct ethhdr *eth, void *data_end,
     return true;
 }
 
```

Comment:
I think its better to add a comment here that SECTION name should start with 'tc' prefix and the reason for the same ?